src/linear_algebra/tensor_product.lean

```diff
@@ -267,6 +267,9 @@ rfl
   x ⊗ₜ (r • y) = r • (x ⊗ₜ[R] y) :=
 (smul_tmul _ _ _).symm
 
+lemma mul_smul_tmul (r s : R) (m : M) (n : N) : (r * s) • (m ⊗ₜ[R] n) = (r • m) ⊗ₜ[R] (s • n) :=
```

Comment:
This lemma would make more sense in reverse, so that it matches `smul_mul_smul`:
```suggestion
lemma smul_tmul_smul (r s : R) (m : M) (n : N) : (r • m) ⊗ₜ[R] (s • n) = (r * s) • (m ⊗ₜ[R] n) :=
```